float64 yaw
float64 velocity_x
float64 velocity_y
float64 velocity_z